### PR TITLE
Contact form pair programming

### PR DESCRIPTION
**Pull request by BenM4k**
In this commit, my collaborator added and HTML form for the contact section plus the footer then I added the styles for all his code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -220,7 +220,53 @@ <h3>Skills</h3>
...
+                    placeholder="Email address" 
+                    required>
+                </li>
+                <li>
+                    <textarea 
+                    name="user_message" 
+                    id="message" 
+                    placeholder="write something..." 
+                    minlength="2" 
+                    maxlength=""
```
Comment (AdnanOlarmmi) [inline index.html:250]: - [ ] According to the requirement the `maxlength` of the `textarea` should be 500, kindly follow suit to enable you deliver what is required of you 😃.

File: index.html [hunk 2]
```diff
@@ -220,7 +220,53 @@ <h3>Skills</h3>
...
+                    <textarea 
+                    name="user_message" 
+                    id="message" 
+                    placeholder="write something..." 
+                    minlength="2" 
+                    maxlength="" 
+                    required></textarea>
+                </li>
+                <li class="form_button flex">
+                    <button type="submit" class="button_enabled">Get in touch</button>
```
Comment (AdnanOlarmmi) [inline index.html:254]: - [ ] Nice one applying the `hover` and `enabled` effects to the button, but it seems the `clicked` effect which is a bit darker than the `enabled` is not been applied, or is it my eye that is deceiving me 🤣.

Kindly fix this, and likewise the `disabled`, to ensure a pixel-perfect design, you can use the [developer's best friend](https://www.google.com/) to make your research.

- Your design
mobile

- Figma design
button2

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @BenM4k and @katfogy, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice design.
- Clean code.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -220,7 +220,53 @@ <h3>Skills</h3>
...
+                    <textarea 
+                    name="user_message" 
+                    id="message" 
+                    placeholder="write something..." 
+                    minlength="2" 
+                    maxlength="500" 
+                    required></textarea>
+                </li>
+                <li class="form_button flex">
+                    <button type="submit" class="button_enabled">Get in touch</button>
```
Comment (LionRouge1) [inline index.html:254]: [OPTIONAL]
- [ ] Kindly remove the contact form from the `footer` tag for semantics reasons.

- A `<footer>` element typically contains:
  - authorship information
  - copyright information
  - contact information
  - sitemap
  - back to top links
  - related documents

**Review by LionRouge1 — APPROVED** [review]
Hi @team, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: index.html (hunk 3 above)
Comment (BenM4k) [reply index.html:254]: Man, really

**Pull request merged**